javascript/javascript2/week2/hyf-bay/main.js

```diff
@@ -0,0 +1,67 @@
+console.log("Script loaded");
+
+const products = getAvailableProducts();
+console.log(products);
+
+const list = document.querySelector("ul");
+let showedProduts = products;
+
+function renderProducts(productArray) {
+    // your code here
+    for (const product of productArray) {
+        const li = document.createElement("li");
+        const header = document.createElement("h2");
+        header.innerHTML = product.name;
+        const price = document.createElement("span");
+        price.innerHTML = "Price: " + product.price;
+        const rating = document.createElement("span");
+        rating.innerHTML = "Rating: " + product.rating;
+        li.appendChild(header);
+        li.appendChild(price)
+        li.appendChild(document.createElement("br"));
+        li.appendChild(rating);
+        list.appendChild(li);
+    }
```

Comment:
Great job, but I suggest to create those elements in `html` file where it will be easier to style them.  Here you can use `querySelector` instead of `createElement`

Reply:
I create `li ` with `header`, `br`, `price`, `rating ` dynamically because I think that`s the easy way to populate it for all products. If I make it in `hmtl `and than use `querySelector` thank I need to copy it to all other products by using `cloneNode` or how is it done?